gui/qt/main_window.py

```diff
@@ -3045,7 +3045,7 @@ def enable_sweep():
                 else:
                     self.show_message(_("User cancelled"))
                     return
-            coins, keypairs = sweep_preparations(keys, self.network)
+            coins, keypairs = sweep_preparations(keys, self.network, sign_schnorr = self.is_schnorr_enabled())
```

Comment:
A bit awkward ... for example if we are sweeping *into* a multisig wallet, why can't we use Schnorr?

Reply:
👍

Reply:
Note that for multisig wallets `is_sign_schnorr` is always False -- I may just add a checkbox to the GUI to permit this as a special case for sweeping INTO multisig if that is the case.  Will add it after this is merged.